go/libcmdline/cmdline.go

```diff
@@ -319,6 +332,18 @@ func (p *CommandLine) PopulateApp(addHelp bool, extraFlags []cli.Flag) {
 			Usage: "Specify an alternate API URI path prefix.",
 		},
 		cli.StringFlag{
+			Name:  "gregor-uri",
```

Comment:
This will be the first time we've used "gregor" in a public facing string? Would it be better to call it `push-server-uri` or something similarly self-explanatory? (On the other hand, will anyone outside of Keybase ever need this flag? Maybe not.)